### ISSUE TITLE
Working example / Demo

### ISSUE THREAD

**Issue by mrbarletta** [4rthem/graphql-mapper#3]
I tried (and failed) to use this project on an existing (and working) graphql-php project, even failed doing it on a new one. 

Is it possible to have a working code? I would help document it more if you give me something to start with.
Thanks @4rthem

**Comment by 4rthem** [comment]
I'm sorry, it may be not enough clear but this is a Work In Progress project as mentioned in the [readme](https://github.com/4rthem/graphql-mapper#work-in-progress), a first proof of concept.

I paused the implementation due to lack of time and because GraphQL specification is still a draft.
But I'm glad to see that I got 10 stars with no communication and that you seem interested to use this mapper. I will reconsider my priorities and try to provide some examples soon.
